client/templates/admin/postEditor/postEditor.html

```diff
@@ -37,8 +37,7 @@
             <textarea
               id="content-input"
               class="article-description"
-              name="content"
-              placeholder="Enter post body content">{{ model.content }}</textarea>
+              name="content">{{ model.content }}</textarea>
```

Comment:
@dagvonkr is already taken care of in `postView.js` `$('#postContent').html(post.content);`